homeassistant/components/bmw_connected_drive/__init__.py

```diff
@@ -223,6 +224,10 @@ def setup_account(
 
     def execute_service(call: ServiceCall) -> None:
         """Execute a service for a vehicle."""
+        _LOGGER.warning(
+            "BMW Connected Drive services are deprecated. Please migrate to the dedicated button entities"
```

Comment:
Adding a link to the updated integration documentation on the Home Assistant website in the warning message would help less advanced users understand what is wrong.

```"BMW Connected Drive services are deprecated and will be removed in a future release. Please migrate to the dedicated button entities. See https://www.home-assistant.io/integrations/bmw_connected_drive/#buttons"```